ee/clickhouse/queries/test/test_trends.py

```diff
@@ -827,3 +829,48 @@ def test_trends_math_without_math_property(self):
             ClickhouseTrends().run(
                 Filter(data={"events": [{"id": "sign up", "math": "sum"}]}), self.team,
             )
+
+    def test_filtering_with_group_props(self):
+        GroupTypeMapping.objects.create(team=self.team, group_type="organization", group_type_index=0)
+        GroupTypeMapping.objects.create(team=self.team, group_type="company", group_type_index=1)
+
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:5", properties={"industry": "finance"})
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:6", properties={"industry": "technology"})
+        create_group(
+            team_id=self.team.pk, group_type_index=1, group_key="company:10", properties={"industry": "finance"}
+        )
+
+        _create_event(
+            event="$pageview",
+            distinct_id="person1",
+            team=self.team,
+            properties={"$group_0": "org:5"},
+            timestamp="2020-01-02T12:00:00Z",
+        )
+        _create_event(
+            event="$pageview",
+            distinct_id="person1",
+            team=self.team,
+            properties={"$group_0": "org:6"},
+            timestamp="2020-01-02T12:00:00Z",
+        )
+        _create_event(
+            event="$pageview",
+            distinct_id="person1",
+            team=self.team,
+            properties={"$group_0": "org:6", "$group_1": "company:10"},
+            timestamp="2020-01-02T12:00:00Z",
+        )
+
+        filter = Filter(
+            {
+                "date_from": "2020-01-01T00:00:00Z",
+                "date_to": "2020-01-12T00:00:00Z",
+                "events": [{"id": "$pageview", "type": "events", "order": 0}],
+                "properties": [{"key": "industry", "value": "finance", "type": "group", "group_type_index": 0}],
+            },
+            team=self.team,
+        )
+
+        response = ClickhouseTrends().run(filter, self.team)
+        self.assertEqual(response[0]["count"], 1)
```

Comment:
Tests here feel incomplete.

Would love to see another one / incorporated into this one, things like:

1. Filtering on multiple group types properties (tests more than one joins query is constructed properly, no param mismatch)
2. ~~... I guess rest come after we have trends querying by unique group~~

Reply:
I agree! 

I'll follow up on this in subsequent PRs :)